src/mono/mono/metadata/sysmath.c

```diff
@@ -219,15 +219,17 @@ ves_icall_System_Math_Log2 (gdouble x)
 }
 
 gdouble
-ves_icall_System_Math_ScaleB (gdouble x, gint32 n)
+ves_icall_System_Math_FusedMultiplyAdd (gdouble x, gdouble y, gdouble z)
 {
-	return scalbn (x, n);
+	return fma (x, y, z);
 }
+#endif
 
+#ifndef ENABLE_NETCORE
 gdouble
-ves_icall_System_Math_FusedMultiplyAdd (gdouble x, gdouble y, gdouble z)
+ves_icall_System_Math_ScaleB(gdouble x, gint32 n)
 {
-	return fma (x, y, z);
+    return scalbn(x, n);
```

Comment:
nit: mono's C code uses tabs for indentation:
```suggestion
	return scalbn(x, n);
```
(same on line 403)